scalability_and_performance/sno-du-connected.adoc

```diff
@@ -0,0 +1,52 @@
+:_content-type: ASSEMBLY
+[id="sno-du-deploying-distributed-units-manually-on-single-node-openshift-connected"]
+= Deploying distributed units manually on single node OpenShift in a connected environment
```

Comment:
@imiller0 @yliu127 Wondering if this title should be changed?

Reply:
My opinion is that this document describes how to tune a cluster for DU workloads and specifically it does not touch on the environment in which that is done. This same procedure holds for both connected and disconnected environments. The closest we get to this being "specific to connected" is that the subscriptions use the default (connected) registry names. I think that we should omit "in a connected environment" here and soften the language in the intro paragraph below. 
@browsell and @kenyis does this sound acceptable to you?

Reply:
@imiller0 Changed the title to "Tuning a cluster for DU workloads on single node OpenShift".

Reply:
I still don't think it rings true.
 How about "Tuning single node OpenShift for a DU workload"

Reply:
@djuran @imiller0 I'll make this change in the next iteration, after QE finishes their review.

Reply:
@djuran @imiller0 Change has been made.

Reply:
LGTM